pkg/api/api.go

```diff
@@ -129,3 +129,15 @@ func requestModePut(r *http.Request) storage.ModePut {
 func requestEncrypt(r *http.Request) bool {
 	return strings.ToLower(r.Header.Get(SwarmEncryptHeader)) == "true"
 }
+
+func (s *server) newTracingHandler(spanTag string) func(h http.Handler) http.Handler {
+	return func(h http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			spanName := fmt.Sprintf("http.%s.%s", r.Method, spanTag)
```

Comment:
I think there is no need for http.$method prefix, basically `spanName := spanTag` is just enough.